redbot/core/core_commands.py

```diff
@@ -4060,7 +4045,7 @@ async def command_manager(self, ctx: commands.Context):
 
     @checks.is_owner()
     @command_manager.command(name="defaultdisablecog")
-    async def command_default_disable_cog(self, ctx: commands.Context, *, cogname: str):
+    async def command_default_disable_cog(self, ctx: commands.Context, *, cogname: CogConverter):
```

Comment:
As with command_name->command, this should also be renamed from cogname to cog:
```suggestion
    async def command_default_disable_cog(self, ctx: commands.Context, *, cog: CogConverter):
```
You will need to update the docstring and docs as well.